docs/DEPLOYMENT_SETUP.md

```diff
@@ -120,6 +120,7 @@ The list of categories can be inferred by looking at the file `.ebextensions/env
 | Variable     | Description                                                                                                  |
 | :----------- | ------------------------------------------------------------------------------------------------------------ |
 | `SSM_PREFIX` | String prefix (typically the environment name) for AWS SSM parameter names to create a .env file for FormSG. |
+| `SECRET_ENV` | String prefix (typically the environment name) used in building of AWS Secrets Manager key.                  |
```

Comment:
nit: the description doesn't seem to be descriptive enough. Suggest "String (typically the environment name) to be used when building AWS Secrets Manager keys in different environments"